pkgs/applications/networking/browsers/chromium/default.nix

```diff
@@ -127,6 +127,10 @@ let
         cp -a ${widevineCdm}/WidevineCdm $out/libexec/chromium/
       ''
     else browser;
+
+  optionalVaapiFlags = lib.optionalString
+    (!enableVaapi)
```

Comment:
Why is this needed? Will not chrome disable by default?

Reply:
No, we changed that behaviour (see the linked PR and issue for the discussion).